policyengine_canada/tests/gov/cra/benefits/oas/oas_net.yaml

```diff
@@ -0,0 +1,10 @@
+- name: Partial repayment
```

Comment:
you can remove this file - we only need unit tests for variables with formulas, not those with only `adds` and `subtracts`